openedx/core/djangoapps/course_groups/tests/test_cohorts.py

```diff
@@ -261,8 +267,8 @@ def test_get_cohort(self):
         )
 
     @ddt.data(
-        (True, 2),
-        (False, 6),
+        (True, 22),
+        (False, 32),
```

Comment:
I'm investigating why these numbers had to become so much larger. I knew they would change a bit, just due to the added overhead of using `CohortMembership`, but an increase of this magnitude seems like it could add performance penalties.